proof/access-control/ARM/ArchRetype_AC.thy

```diff
@@ -33,12 +33,25 @@ end
 
 context retype_region_proofs' begin interpretation Arch .
 
+\<comment> \<open>FIXME: move to pas_refined_state_objs_to_policy_subset?\<close>
```

Comment:
If the proof impact is minimal, that would be ideal. Curious what case you ran into where the subset relation was too strong.

Reply:
The subset relation is on `domains_of_state`, which retyping now could change. I needed to use the subjects from `tcb_domain_map_wellformed` to show that in this case it still doesn't affect `pas_refined`.